src/components/Icon.stories.tsx

```diff
@@ -22,6 +22,7 @@ export const Icon = (props: IconProps) => {
     "x",
     "loader",
     "linkExternal",
+    "bxLink",
```

Comment:
**nit:** Can we call this something else besides `bxLink`, maybe just `link`? The `bx` prefix comes with the Box icon library set design uses.
Also, can you ask the designer for this to add the icon to Beam's Icon Pages in Figma? https://www.figma.com/file/ePfj5WZKtGfOgABpZcUtRE/%E2%9C%A8Beam-Icons?node-id=5-14&t=RlW8BvpXJ1ucYGU4-0